donkeycar/parts/controller.py

```diff
@@ -659,6 +660,13 @@ def __init__(self, poll_delay=0.0,
         self.axis_trigger_map = {}
         self.init_trigger_maps()
 
+        self.shiftKeyHistory=[]
```

Comment:
Please follow code formatting as it is presented in lines above (spaces around =)

Reply:
Hi, i have updated all occurences  and added spaces before and after '='. 
thx.